### PR TITLE
path changes for M1 Mac

### PR DESCRIPTION
**Pull request by olivierbouwman**

### CODE REVIEW

File: environment_setup.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ if confirmupdate "Would you like to install local development programs like PHPS
   # Set some standard git configuration
   git config --global pull.rebase true
 
-  export PATH=./vendor/bin:~/.composer/vendor/bin:/usr/local/bin:/usr/local/sbin:$PATH
+  export PATH=./vendor/bin:~/.composer/vendor/bin:$PATH
```
Comment (olivierbouwman) [inline environment_setup.sh]: These no longer exist for me so assuming we don't need them anymore

Comment (wxactly) [reply environment_setup.sh]: @olivierbouwman - I believe we still need `/opt/homebrew/bin` in our path - do you see brew binaries there on your machine?

**Review by wxactly — CHANGES_REQUESTED** [review]

**Comment by BrendanJercich** [comment]
@wxactly ran into an onboarding challenge today that having this in main would have solved! I have unilaterally updated the branch to follow your recommendation.

**Review by wxactly — APPROVED** [review]

**Comment by mariacha** [comment]
And I ran into all the same problems, and am merging!

**Pull request merged**